system-tests/test/testConfigOverrides_spec.ts

```diff
@@ -61,7 +61,7 @@ describe('testConfigOverrides', () => {
       spec: 'testConfigOverrides/invalid.js',
       snapshot: true,
       browser: browserList,
-      expectedExitCode: 8,
+      expectedExitCode: 14,
```

Comment:
dare I ask what exit code `14` is?

Reply:
Yup! there are 14 failing tests in the invalid.js file